src/models/apiLogs.model.ts

```diff
@@ -25,6 +25,12 @@ const apiLogSchema = new Schema({
     statusCode: { type: Number },
     errorMessage: { type: String },
```

Comment:
as discussed this field should be removed
and response body needs to be added and captured